media/com_redshop/js/redshop.registration.js

```diff
@@ -128,7 +129,16 @@ if (typeof(window['jQuery']) != "undefined") {
                 },
                 zipcode: {
                     zipcodeValidation: true,
-                    required: redSHOP.RSConfig._('REQUIRED_POSTAL_CODE') == 1 ? true : false
+                    required: redSHOP.RSConfig._('REQUIRED_POSTAL_CODE') == 1 ? true : false,
+                    minlength: 9,
```

Comment:
This is wrong. In Denmark use 4 digits zip code.

@lunguyenhat

Reply:
yes, it should get the value from configurations, not the hard code here @Rmh1978